### DISCUSSION TITLE
How to enable MULTISIG?

### DISCUSSION THREAD

**Discussion by h-sigma** [btcpayserver/btcpayserver#3562] [Technical Support]
I'm fairly new to bitcoin (being an ethereum user). I saw in [one of the FAQs](https://docs.btcpayserver.org/FAQ/Wallet/#what-is-a-derivation-scheme) that it is possible to use a derivation scheme in addition to an xPub to enable MULTISIG payments. In my specific use-case, I want to split payments (1 input 2 outputs) so I can charge users of my instance for very large transactions in btc. Of course, I have no idea how to set this up inside btcpay as the FAQ section was quite short. Any help from some pointers to a full fledged guide would be helpful.

In the meantime I'll be slogging through [bip 32](https://github.com/bitcoin/bips/blob/master/bip-0032.mediawiki) to better under the process.

**Comment by Kukks** [comment]
Your request has nothing to do with multisig.

**Reply by mp21-org** [reply]
@h-sigma kindly check out the [BTCPay Transmuter](https://github.com/btcpayserver/btcTransmuter) project for setting up fee splitting. It involves a lot of configuration and is still considered #reckless technology. But we do have a solution for what you want.

**Comment by pavlenex** [comment]
https://docs.btcpayserver.org/Multisig/#step-4-send-a-transaction-from-the-multisig-wallet
